Godeps/_workspace/src/github.com/prometheus/client_golang/model/labelname.go

```diff
@@ -46,6 +46,10 @@ const (
 	// will not be attached to time series.
 	MetaLabelPrefix = "__meta_"
 
+	// ParamLabelPrefix is a prefix for labels that provide URL parameters
+	// used to scrape a target.
+	ParamLabelPrefix = "__param_"
```

Comment:
Cool! Could you also update the commit rev in https://github.com/prometheus/prometheus/blob/master/Godeps/Godeps.json#L45-L64 to `3a499bf7fc46bc58337ce612d0cbb29c550b8118` now? (and probably remove the comment since it's not a release version anymore).